test.util/src/main/java/org/kaazing/test/util/MethodExecutionTrace.java

```diff
@@ -32,49 +36,55 @@
     @Rule
     public MethodRule testExecutionTrace = new MethodExecutionTrace();
  */
-public class MethodExecutionTrace extends TestWatchman {
-    //if we ever want the full test name (including method),
-    //we can call the public static method which retrieves this
-    //variable. This variable is set in the starting method, and
-    //and unset in the fail/success methods
-    private static String currentFullTestName;
-
-    public static String getFullTestName() {
-        return currentFullTestName;
-    }
+public class MethodExecutionTrace extends TestWatcher {
 
     /**
      * Use this constructor to set up a particular log4j configuration using a properties file,
-     * e.g. to use MemoryAppender.
+     * or null if you do not want to load any log4j configuration properties.
      * See gateway.server src/test/resources/log4j-trace.properties for an example.
      * @param log4jConfigPropertiesFileName
      */
-    public MethodExecutionTrace(String log4jConfigPropertiesFileName) {
+    public MethodExecutionTrace(String log4jPropertiesResourceName) {
         MemoryAppender.initialize();
```

Comment:
I never really liked the idea of having log4j properties files all over the place and I really like this change.  However, I was curious if you considered exposing the [programatic configuration](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/BasicConfigurator.html) of the log4j.   I'm not familiar with the API or how easy it is to use but it MAY be easier to customize logging programmatically rather than adding a log4j resource file.